### DISCUSSION TITLE
sealos_4.2.1-rc4_linux_amd64.tar.gz私有云部署没有WEB UI界面吗？

### DISCUSSION THREAD

**Discussion by 1186118500** [labring/sealos#3272] [Q&A] [answered]
sealos_4.2.1-rc4_linux_amd64.tar.gz私有云部署没有WEB UI界面吗？
这个怎么部署呢？有详细步骤吗？部署完有WEB UI 界面吗？

**Comment by zzjin** [comment]
Hello, sealos with web ui will have it's own cluster-image to install with one command.
We are now testing it and will have document and image online asap.

**Reply by gclm** [reply]
预计还需要多久才能整理出来呢？如果我想自己尝试构建web ui,有什么提供思路或者简略文档呢

**Reply by zzjin** [reply]
Hello, for independent deployment, you can refer to the deploy directory of each component under `controller`/`desktop` dirs, which contains the packaging of cluster images and parameter configuration. 

The compiled all-in-one cluster image has under testing.

**Reply by gclm** [reply]
hi，目前web ui 进度如何。预计什么时候可以上线

**Reply by bxy4543** [reply, marked as answer]
https://github.com/labring/sealos/tree/main/deploy/cloud
&&
Community User Practice: https://mp.weixin.qq.com/s/w6Wz71-H8OLY1HVzYfRCwQ
